tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
I started with the one in pilot - but I don't think it's safe (or good idea) to attempt to unify, quite the opposite.